docs/getting-started/windows-manual.md

```diff
@@ -72,3 +84,18 @@ Start-Service splunk-otel-collector
 ```
 
 The collector logs and errors can be viewed in the Windows Event Viewer.
+
+### Docker
+
+Create build of otel collector on local system.
+```bash
+$ git clone https://github.com/signalfx/splunk-otel-collector.git
+$ cd splunk-otel-collector
+$ docker build -t otelcol --build-arg SMART_AGENT_RELEASE=5.11.2 -f .\cmd\otelcol\Dockerfile.windows .\cmd\otelcol\
```

Comment:
Probably don't need this section unless we just want to document some local dev steps.  But if we leave it in, need to add a step to either build the exe or download a pre-built exe and copy it to `.\cmd\otelcol\otelcol.exe` before `docker build`.

Reply:
Ok removed it.